functions-app/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "GPL-3.0",
   "scripts": {
     "start": "func host start --port 7072",
-    "test": "yarn lint && yarn run nyc -r html jasmine **/*.spec.js */**/*.spec.js",
+    "test": "yarn lint && yarn run nyc -r html jasmine **/*.spec.js */**/*.spec.js */**/**/*.spec.js",
```

Comment:
would `**/*.spec.js` not cover all your subdirectories in this particular case or am i overlooking something subtle?

Reply:
It was needed as due to the extra level of sub-directory nesting with the 'detections' directory.  So, apparently not.

Reply:
that's odd.  i was under the impression that the first one `**/*.js` covers all recursively